lib/schema_generator/format.rb

```diff
@@ -267,7 +267,7 @@ def determined_definition
 
     class Links
       ALLOWED_KEYS = %w(description required minItems maxItems).freeze
-      LINKS_WITHOUT_BASE_PATHS = %w(ordered_contacts ordered_foi_contacts world_locations).freeze
+      LINKS_WITHOUT_BASE_PATHS = %w(facet_groups facet_values ordered_contacts ordered_foi_contacts world_locations).freeze
```

Comment:
Shouldn't `facet` be in this list as well?

Reply:
@leenagupte good point - added.